samples/csharp/MicrosoftGraphSDK/Assignment.cs

```diff
@@ -32,17 +32,41 @@ public class Assignment
             }
             catch (Exception ex)
             {
-                throw new GraphException($"GetAssignment call: {ex.Message}", classId, assignmentId);
+                throw new GraphException($"GetAssignmentAsync call: {ex.Message}", ex, classId, assignmentId);
+            }
+        }
+
+        /// <summary>
+        /// Get all the assignments from the class
+        /// </summary>
+        /// <param name="client">Microsoft Graph service client</param>
+        /// <param name="classId">User class id</param>
+        /// <returns>IEducationClassAssignmentsCollectionPage</returns>
+        public static async Task<IEducationClassAssignmentsCollectionPage> GetAssignmentsAsync(
+            GraphServiceClient client,
+            string classId)
```

Comment:
add input validation across the PR for method inputs e.g., string should not be null or empty

Reply:
updated input checks in the workflow